src/Element_TEST.cc

```diff
@@ -639,12 +655,16 @@ TEST(Element, Copy)
   EXPECT_EQ("/sdf/world[@name=\"default\"]", dest->XmlPath());
   EXPECT_EQ("1.5", dest->OriginalVersion());
 
+  dest->SetName("Copied");
+
   sdf::ParamPtr param = dest->GetValue();
   ASSERT_TRUE(param->IsType<std::string>());
   ASSERT_EQ(param->GetKey(), "test");
   ASSERT_EQ(param->GetTypeName(), "string");
   ASSERT_EQ(param->GetDefaultAsString(), "val");
   ASSERT_EQ(param->GetDescription(), "val description");
+  ASSERT_NE(param->GetParentElement(), nullptr);
```

Comment:
Can you add an equality expectation on the `GetParentElement()` pointer and `dest`? 
```suggestion
  ASSERT_NE(param->GetParentElement(), nullptr);
  EXPECT_EQ(param->GetParentElement(), dest);
```

Reply:
That makes much more sense than just comparing the names, thanks! https://github.com/ignitionrobotics/sdformat/pull/589/commits/4687c0e3014cf5dc8d58670cf328a1022bc1335e